src/lib/util/webhook.ts

```diff
@@ -0,0 +1,150 @@
+import type { rehearsal } from '$lib/types/domain/rehearsal'
+import type { announcement } from '$lib/types/domain/announcement'
+import { PUBLIC_DISCORD_WEBHOOK_URL } from '$env/static/public'
+
+const colours: string[] = ['00b380', '800020', '696969']
+
+export async function newRehearsalPost(rehearsal: rehearsal) {
```

Comment:
je bent best wel veel dingen in params aan het herhalen. Misschien ff een sendBMNWebhookMessage functie definieren die alleen de dingen neemt die veranderen, en dingen zoals kleur, avatar, author etc voor je zet. Scheelt weer code duplicatie

Reply:
Of het aan sendWebhookMessage toevoegen